File: dev/requirements/screen/cases/card-footnotes.4.5.case.swift
#if canImport(SwiftUI) && os(iOS)
import SetlistUI

/// 4.5 — each card's footnote: a run time, the live line, an estimate.
let cardFootnotes_4_5 = RequirementCase(id: "4.5") {
    try MainActor.assumeIsolated {
        try expectScreen(StageScreen(state: StageFixture.state()), slug: "card-footnotes", id: "4.5")
    }
}
#endif

File: dev/requirements/screen/cases/card-footnotes.4.5.case.dart
import 'package:setlist_ui/setlist_ui.dart';

import '../../shared/requirement_case.dart';
import '../../shared/snapshot.dart';

/// 4.5 — Each card's footnote states its state in time.
final cardFootnotes_4_5 = RequirementCase('4.5', (tester) async {
  await expectRegion(
    tester,
    StageFixture.state(),
    region: StageRegion.grid,
    slug: 'card-footnotes',
    id: '4.5',
  );
});
